src/test/java/com/powsybl/openloadflow/ac/DistributedSlackOnLoadTest.java

```diff
@@ -83,4 +88,21 @@ void testWithLoadDetail() {
         assertActivePowerEquals(12.727, l5.getTerminal());
         assertActivePowerEquals(-50, l6.getTerminal()); // same as p0 because p0 < 0
     }
+
+    @Test
+    void testPowerFactorConstant() {
+        // given
+        Network testNetwork = Importers.loadNetwork(Paths.get("src", "test", "resources", "2.xiidm"));
```

Comment:
Use `Network network = EurostagTutorialExample1Factory.create();` instead.

Reply:
And thus delete the `2.xiidm` file.